src/bindings/dom.js

```diff
@@ -281,6 +281,20 @@ export default class LocalizationObserver {
     return this.translateRootContent(root).then(setLangs);
   }
 
+  translateRootContent(root) {
+    markStart();
+
+    const anonChildren = document.getAnonymousNodes ?
+      document.getAnonymousNodes(root) : null;
+    if (!anonChildren) {
+      return this.translateFragment(root).then(markEnd);
+    }
```

Comment:
I think there was a reason why we wanted to translate both the anonymous content attached to the root _as well as_ the root itself.  I don't remember exactly right now btu if we want to change the logic here I'd suggest filing a separate bug.